requirements.txt

```diff
@@ -1,3 +1,4 @@
 # Pin pyyaml to 5.3.1 until yaml/pyyaml#724 is fixed.
 PyYAML==5.3.1
-gevent==22.10.2
+gevent==22.10.2; python_version <= '3.8'
+gevent==24.11.1; python_version > '3.8'
```

Comment:
Does `pip3` from the CentOS 7 distribution support this syntax?

Reply:
I've tested it with yum python3 package and everything seems to work.

Reply:
Nice, thanks!